packages/create-schemas/src/utils.ts

```diff
@@ -0,0 +1,69 @@
+import { isAbsolute, join } from "node:path";
+import { fileURLToPath, pathToFileURL } from "node:url";
+
+function isFileURLAsText(input: string | URL): input is `file://${string}` {
```

Comment:
nit: `isFileUrlAsText` may be more readable? or is that an expected TS standard to use URL?

Reply:
It's an old web convention. I think it originated from an internal naming convention at Microsoft or Mozilla that stuck around in web APIs. It went:

- 3 letter acronyms and less: uppercase it.
- 4 letter acronyms and more: normal case.

This is how we have ended up with APIs like [XMLHttpRequest](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest), where `XML` is uppercased but `Http` isn't. By convention, `URL` is also uppercased, like in the [URL class](https://developer.mozilla.org/en-US/docs/Web/API/URL).